### ISSUE TITLE
Cannot link on Ubuntu 21.04

### ISSUE THREAD

**Issue by lukoml** [anyks/asc#4]
`/usr/bin/ld: cannot find -ltcmalloc`

```
$ apt search tcmalloc
Sorting... Done
Full Text Search... Done
libtcmalloc-minimal4/hirsute,now 2.9.1-0ubuntu1 amd64 [installed,automatic]
  efficient thread-caching malloc
```

```
$ locate tcmalloc
/usr/lib/x86_64-linux-gnu/libtcmalloc_minimal.so.4
/usr/lib/x86_64-linux-gnu/libtcmalloc_minimal.so.4.5.9
/usr/lib/x86_64-linux-gnu/libtcmalloc_minimal_debug.so.4
/usr/lib/x86_64-linux-gnu/libtcmalloc_minimal_debug.so.4.5.9
...
```
Dependency on tcmalloc is not detected by cmake.

**Comment by anyks** [comment]
I will fix it in a new commit

**Comment by anyks** [comment]
Fixed in version v1.2.5
